### DISCUSSION TITLE
Openssl build in conan perl issue

### DISCUSSION THREAD

**Discussion by way4sahil** [openssl/openssl#24089] [Q&A] [answered]
conan install openssl/1.1.1t@ --build=missing -pr=rededger_owa5x

penssl/1.1.1t: ['"conan-Release-Linux-armv8-gcc-11"', 'no-shared', '--prefix=/', '--openssldir="/etc/ssl"', 'no-unit-test', 'threads', 'PERL=perl', 'no-tests', '--release', '--libdir=lib', '-fPIC', 'no-md2']

./Configure: Perl lib version (5.34.0) doesn't match executable '/home/ihn8kor/.conan/data/toolchain-aarch64-rededgers/3.0/_/_/toolchain/sysroots/x86_64-fslcsdk-linux/usr/bin/perl.real' version (5.34.1) at /usr/lib/x86_64-linux-gnu/perl/5.34.0/Config.pm line 62.

Compilation failed in require at ./Configure line 14.

BEGIN failed--compilation aborted at ./Configure line 14.

openssl/1.1.1t: 

openssl/1.1.1t: ERROR: Package '115c8fa606e08a868a5ec8073c26c1a096c7263b' build failed

openssl/1.1.1t: WARN: Build folder /home/ihn8kor/.conan/data/openssl/1.1.1t/_/_/build/115c8fa606e08a868a5ec8073c26c1a096c7263b/build-release

ERROR: openssl/1.1.1t: Error in build() method, line 723

	self.run(f'{self._perl} ./Configure {args}')

	ConanException: Error 255 while executing perl ./Configure "conan-Release-Linux-armv8-gcc-11" no-shared --prefix=/ --openssldir="/etc/ssl" no-unit-test threads PERL=perl no-tests --release --libdir=lib -fPIC no-md2

**Comment by t8m** [comment, marked as answer]
You have some problem with the perl used in the Conan build environment. Unfortunately I cannot help you much more with it.
